### PR TITLE
Add Diff v3 backport

### PR DESCRIPTION
**Pull request by keradus**

### CODE REVIEW

**Comment by keradus** [comment]
ping @SpacePossum , please decide ;)

File: src/v1_4/Differ.php [hunk 1]
```diff
@@ -232,7 +232,7 @@ public function diffToArray($from, $to, LongestCommonSubsequence $lcs = null)
 
         if ($this->detectUnmatchedLineEndings($fromMatches, $toMatches)) {
             $diff[] = array(
-                '#Warning: Strings contain different line endings!',
```
Comment (SpacePossum) [inline src/v1_4/Differ.php:235]: @keradus why was change needed for v3.0?

File: src/v2_0/Differ.php [hunk 2]
```diff
@@ -158,7 +158,7 @@ public function diffToArray($from, $to, LongestCommonSubsequenceCalculator $lcs
         }
 
         if ($this->detectUnmatchedLineEndings($diff)) {
-            \array_unshift($diff, ["#Warning: Strings contain different line endings!\n", 3]);
```
Comment (SpacePossum) [inline src/v2_0/Differ.php:161]: @keradus same here, not sure why this change is needed for v3.0?

File: src/v1_4/Differ.php (hunk 1 above)
Comment (keradus) [reply src/v1_4/Differ.php:235]: i copy-pasted v1 from original repo as well, that was the change from there

File: src/v2_0/Differ.php (hunk 2 above)
Comment (keradus) [reply src/v2_0/Differ.php:161]: i copy-pasted v2 from original repo as well, that was the change from there

**Comment by SpacePossum** [comment]
thanks @keradus ! :D

**Comment by keradus** [comment]
new release? :>

**Comment by SpacePossum** [comment]
v1.3.0 is done ^_^

**Comment by keradus** [comment]
Thanks !
